### PR TITLE
fix: Improve horizontal likes fragment

### PR DESCRIPTION
**Pull request by anhanh11001**
Details:
Add horizontal views on Favorite Fragment to improve current view

Fixes: #1493

Screenshots for the change:
55633880-436298290475818-8601806044814901248-n-2

### CODE REVIEW

File: app/src/main/res/layout-land/fragment_favorite.xml [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+                   android:text="@string/find_something_to_do"
+                   android:textColor="@color/black"
+                   android:id="@+id/find_text"
+                   app:layout_constraintLeft_toLeftOf="parent"
+                   android:layout_marginLeft="@dimen/view_height_25dp"
+                   app:layout_constraintTop_toTopOf="parent"
+                   app:layout_constraintBottom_toBottomOf="parent"/>
+
+               <TextView
+                   android:id="@+id/today_button"
```
Comment (addiegupta) [inline app/src/main/res/layout-land/fragment_favorite.xml]: Use camelCase

**Comment by iamareebjamal** [comment]
No need to duplicate the whole layout, use dimens-land

**Comment by anhanh11001** [comment]
@iamareebjamal the current vertical view use ConstraintLayout as the rootView, so in the horizontal variations, there are changes in constraints, not just in dimensions so that's why I'm using a new layout. It's not really a duplicated one

**Comment by iamareebjamal** [comment]
Why can't we use the same layout and make it responsive?

**Comment by anhanh11001** [comment]
I will improve this

**Comment by anhanh11001** [comment]
@iamareebjamal actually I'm having some difficulty on this as I haven't come up any ideas on how to contain display current vertical screen (3 TextView - today, tomorrow, next week - in the first line and the last one - next month - in the second line) and put all 4 TextView in one line in horizontal screen with just one XML file. Clearly I cannot put all 4 text views in one line in vertical screen. Can you give me some advice?

Current view:

**Comment by iamareebjamal** [comment]
Try LinearLayout

**Comment by anhanh11001** [comment]
@iamareebjamal that's what I was referring to. Putting 4 TextView in one LinearLayout in the vertical screen, the view looks really compact and ugly. But I cannot build the current vertical view with just a simple linear layout as Linear Layout doesn't support moving an Android View to the next line when there is not enough space.

**Comment by iamareebjamal** [comment]
It supports wrapping, if it cannot, use flex layout

**Comment by aman210697** [comment]
> @iamareebjamal that's what I was referring to. Putting 4 TextView in one LinearLayout in the vertical screen, the view looks really compact and ugly. But I cannot build the current vertical view with just a simple linear layout as Linear Layout doesn't support moving an Android View to the next line when there is not enough space.

1. Use MaterialButton
2. Wrap textViews inside ChipGroup, it'll handle.

**Comment by anhanh11001** [comment]
@aman210697 thanks, seems like a way to go

**Comment by anhanh11001** [comment]
UPDATE: 
Detail:
- Use Chip & ChipGroup to make the button for today/tomorrow/weekend/next month instead of TextView
- ChipGroup automatically reset the view to wrap the content correctly

**Pull request merged**